src/components/Login/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { signInWithEmailAndPassword, signOut } from "firebase/auth";
+import { firebaseAuth } from "../../../firebaseConfig";
+import { useState } from "react";
+import { StyleSheet, View, Text, TextInput, Button } from "react-native";
+import useUser from "@/hooks/useUser";
+
+export default function Login() {
+  const [email, useEmail] = useState("");
+  const [password, usePassword] = useState("");
+  const [error, useError] = useState("");
```

Comment:
The second argument returned from useState should be referred to with setX not useX.
So it should be setEmail, setUser, setError, etc.